adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -0,0 +1,143 @@
+//
+//  GADPangleRTBNativeRenderer.m
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import "GADPangleRTBNativeRenderer.h"
+#import "GADMediationAdapterPangleConstants.h"
+#import <BUAdSDK/BUAdSDK.h>
+#import "GADMAdapterPangleUtils.h"
+
+static NSString *const BUDNativeAdTranslateKey = @"bu_nativeAd";
+@interface GADPangleRTBNativeRenderer()<BUNativeAdDelegate>
```

Comment:
Ditto, as native ad is currently not supported, can you save the code on your end and remove this file?